server/sandbox_run_linux.go

```diff
@@ -42,6 +42,12 @@ import (
 // DefaultUserNSSize is the default size for the user namespace created
 const DefaultUserNSSize = 65536
 
+const (
+	kubeletPodsRootDir    = "/var/lib/kubelet/pods"
```

Comment:
probably a note somewhere should specify this only supports the default kubelet root and there's no way to configure it differently atm